app/javascript/src/NewService/components/FormElements/ServiceManualListItems.tsx

```diff
@@ -36,7 +36,7 @@ const ServiceManualListItems: FunctionComponent<Props> = ({
         />
         <input id="service_system_name" maxLength={255} name="service[system_name]" type="text" value={systemName} onChange={onChange(setSystemName)} />
         <p className={`${errors.system_name ? 'inline-errors' : 'inline-hints'}`}>
-          Only ASCII letters, numbers, dashes and underscores are allowed.
+          { errors.system_name ? errors.system_name : 'Only ASCII letters, numbers, dashes and underscores are allowed.' }
```

Comment:
Here we're removing the helper text (`'Only ASCII letters, numbers, dashes and underscores are allowed.'`) when there's an error. What we'd need is to maintain that string as an `inline-hints` and add the `inline-errors` when necessary.

Reply:
Can you check the screenshots I added

Reply:
@lvillen  This is what you are asking and is updated.

now the errror shows exactly like back end also in flash message same error was coming so updated that also like backend so wont show similar error twcs
![image](https://user-images.githubusercontent.com/84672731/228840937-8d20a05d-b20f-4c64-9a2e-8d36d0b8f1f1.png)